src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -310,10 +321,12 @@ std::string tenantFromDb(std::string& database)
 {
   std::string r;
   std::string prefix  = dbPrefix + "-";
+
   if (strncmp(prefix.c_str(), database.c_str(), strlen(prefix.c_str())) == 0)
   {
     char tenant[MAX_SERVICE_NAME_LEN];
```

Comment:
I think I did that ... Perhaps in one of the many branches during the 'cache-crisis' and it is not here ...
Yes, important to change that 1024 to ... 50 was it?
Let's include this change in this PR.